tests/workspaces/local_workspace_cache_remove_test.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from tango import Step
+from tango.common.testing import TangoTestCase
+from tango.step_info import StepState
+from tango.workspaces import LocalWorkspace
+
+
+class AdditionStep(Step):
+    def run(self, a: int, b: int) -> int:  # type: ignore
+        return a + b
+
+
+class Test_Cache_Remove_Workspace(TangoTestCase):
```

Comment:
```suggestion
class TestLocalWorkspace(TangoTestCase):
```

General class name, also convention dictates no underscores in class names.